chainercv/visualizations/vis_instance_segmentation.py

```diff
@@ -67,10 +74,15 @@ def vis_instance_segmentation(
         raise ValueError('The length of score must be same as that of bbox')
 
     n_inst = len(bbox)
-    colors = np.array([_default_cmap(l) for l in range(1, n_inst + 1)])
+    if colors is None:
+        colors = [voc_colormap(l) for l in range(1, n_inst + 1)]
+    colors = np.array(colors)
 
     canvas_img = img.transpose((1, 2, 0)).copy()
```

Comment:
For visualization, I think it is better to overlap instance segmentation images as `vis semantic segmentation`.
`canvas_img` makes this function less useful.

Reply:
Sorry. I do not follow why `vis semantic segmentation` comes up?
Can you elaborate on this point more?

Reply:
I mean how about reverting https://github.com/chainer/chainercv/commit/f7a41e4ddaa12a5cb19d0458431e88a16edd10b6?
the visualization function can be overlapped together with `img=None`.

Reply:
I mean use `vis_image` in `vis_instance_segmentation`, and visualize `canvas_img` on the image.
It will be useful when we support `img=None`.

Reply:
I will support `img=None` in this PR.

Reply:
Great! thank you :)